modules/oe_content_person/src/Plugin/ConceptSubset/HumanSexPerson.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_person\Plugin\ConceptSubset;
+
+use Drupal\Core\Entity\Query\QueryInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\rdf_entity\RdfFieldHandlerInterface;
+use Drupal\rdf_skos\ConceptSubsetPluginBase;
+use Drupal\rdf_skos\Plugin\PredicateMapperInterface;
+
+/**
+ * Human sex subset for person gender.
+ *
+ * @ConceptSubset(
+ *   id = "oe_content_person_human_sex_person",
```

Comment:
`oe_content_gender`. And let's move move this to oe_content as it will probably be used everywhere the human sex vocab will be used.